browser/src/control/Control.UIManager.js

```diff
@@ -51,10 +51,49 @@ L.Control.UIManager = L.Control.extend({
 			|| forceCompact === false;
 	},
 
+	// Dark mode toggle
+
+	loadLightMode: function() {
+		document.documentElement.setAttribute('data-theme','light');
+		this.map.fire('darkmodechanged');
+	},
+
+	loadDarkMode: function() {
+		document.documentElement.setAttribute('data-theme','dark');
+		this.map.fire('darkmodechanged');
+	},
+
+	getDarkModeState: function() {
+		return this.getSavedStateOrDefault('colorTheme', false);
+	},
+
+	toggleDarkMode: function() {
+		var initialScheme = this.getDarkModeState();
+		if (initialScheme) {
+			this.setSavedState('colorTheme',false);
```

Comment:
`colorTheme` value being boolean is a bit confusing when it will be used out side these functions.

you can either change it to `darkTheme` or change value to string.